src/main/java/com/github/manolo8/darkbot/core/manager/StatsManager.java

```diff
@@ -243,4 +256,63 @@ public double getTotalHonor() {
     public double getEarnedHonor() {
         return earnedHonor;
     }
+
+    public AverageStats getCpuStats() {
+        return cpuStat;
+    }
+
+    public AverageStats getPingStats() {
+        return pingStat;
+    }
+
+    public AverageStats getTickStats() {
+        return tickStat;
+    }
+
+    public AverageStats getMemoryStats() {
+        return memoryStat;
+    }
+
+    public static class AverageStats {
+        private static final DecimalFormat MAX_TWO_PLACES_FORMAT = new DecimalFormat("0.##");
+
+        private int count;
+        private double last, average, min = Double.MAX_VALUE, max = Double.MIN_VALUE;
+
+        private Consumer<String> onChange;
+
+        public void accept(double value) {
+            max = Math.max(max, value);
+            min = Math.min(min, value);
+
+            if (last != value && onChange != null)
+                onChange.accept(MAX_TWO_PLACES_FORMAT.format(value));
+            last = value;
+
+            if (count == Integer.MAX_VALUE) {
+                count = 0;
+                average = 0;
+            }
```

Comment:
If we're thinking it's even possible that we'll reach int max value, just use a long instead